### PR TITLE
Kavansabnani patch 8

### PR DESCRIPTION
**Pull request by kavansabnani**
SCD changes on Sending User Profiles and Sending Group Profiles pages. Also a minor formatting change for Mirror on the Sending Events page

### CODE REVIEW

File: pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx [hunk 1]
```diff
@@ -20,6 +20,26 @@ Once you have created a warehouse source, follow the below steps to send user pr
...
+
+Profile History allows you to point Mixpanel at a group table formatted as [Slowly Changing Dimensions Type 2](https://en.wikipedia.org/wiki/Slowly_changing_dimension#Type_2:_add_new_row). This allows you to set up reports to analyze the historic values of group properties, in addition to latest value.
+
+Setup instructions:
+1. When creating a new Group Table, there will be a Type option. Select History Table.
+2. Map the Group Key to the appropriate group, Group ID to the Group ID column and Start Time to the Start Time column.
+3. You will only have the option for a Time Based sync for a History Table. See below for further setup information.
+
+
+**Note:** For billing purposes each update in the SCD Type 2 table are processed as events.
```
Comment (sowmya17a) [inline pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx]: super-nit: is processed as an event.

File: pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx [hunk 2]
```diff
@@ -20,6 +20,26 @@ Once you have created a warehouse source, follow the below steps to send user pr
...
+  Profile History is in early access and not available on all projects. Request access
+  <a href="https://forms.gle/x8mbU6FVe5uHiVXF6">here</a>.
+</Callout>
+
+Profile History allows you to point Mixpanel at a group table formatted as [Slowly Changing Dimensions Type 2](https://en.wikipedia.org/wiki/Slowly_changing_dimension#Type_2:_add_new_row). This allows you to set up reports to analyze the historic values of group properties, in addition to latest value.
+
+Setup instructions:
+1. When creating a new Group Table, there will be a Type option. Select History Table.
+2. Map the Group Key to the appropriate group, Group ID to the Group ID column and Start Time to the Start Time column.
+3. You will only have the option for a Time Based sync for a History Table. See below for further setup information.
```
Comment (sowmya17a) [inline pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx:37]: Should we mention something about how updates/corrections/reconciliations work ie via an "updated_at" timstamp column?

**Review by sowmya17a — APPROVED** [review]

File: pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx (hunk 2 above)
Comment (kavansabnani) [reply pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx:37]: that's covered in the sync option section below

File: pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx (hunk 1 above)
Comment (kavansabnani) [reply pages/docs/tracking-methods/data-warehouse/sending-group-profiles.mdx]: i'll correct this, ty

**Comment by kavansabnani** [comment]
@ishamehramixpanel i don't have access to Vercel to see why this is failing. Can you help please?

**Review by aswani521 — APPROVED** [review]
LGTM

**Review by sowmya17a — APPROVED** [review]

**Pull request merged**